org.free_astro.siril.json

```diff
@@ -223,11 +223,67 @@
                 "/include",
                 "/lib/pkgconfig"
             ],
+			"modules": [
+			    {
+					"name": "ffmpeg",
+					"config-opts": [
+						"--disable-static",
+						"--enable-shared",
+						"--enable-pic",
+						"--disable-avdevice",
+						"--disable-postproc",
+						"--disable-swscale",
+						"--disable-programs",
+						"--disable-ffplay",
+						"--disable-ffprobe",
+						"--disable-ffmpeg",
+						"--disable-encoder=flac",
+						"--disable-protocols",
+						"--disable-devices",
+						"--disable-network",
+						"--disable-hwaccels",
+						"--disable-dxva2",
+						"--disable-vdpau",
+						"--disable-filters",
+						"--enable-filter=yadif",
+						"--disable-doc",
+						"--disable-d3d11va",
+						"--disable-dxva2",
+						"--disable-audiotoolbox",
+						"--disable-videotoolbox",
+						"--disable-vaapi",
+						"--disable-crystalhd",
+						"--disable-mediacodec",
+						"--disable-nvenc",
+						"--disable-mmal",
+						"--disable-omx",
+						"--disable-omx-rpi",
+						"--disable-cuda",
+						"--disable-cuvid",
+						"--disable-libmfx",
+						"--disable-libnpp",
+						"--disable-iconv",
+						"--disable-jni",
+						"--disable-v4l2_m2m",
+						"--enable-gpl",
+						"--enable-optimizations"
+					],
+					"sources": [
+						{
+							"type": "git",
+							"url": "https://github.com/FFmpeg/FFmpeg.git",
+							"tag": "n6.0",
+							"commit": "ea3d24bbe3c58b171e55fe2151fc7ffaca3ab3d2"
+						}
+					]
+				}
+			],
             "sources": [
                 {
-                    "type": "archive",
-                    "url": "https://github.com/FFMS/ffms2/archive/2.40.tar.gz",
-                    "sha256": "82e95662946f3d6e1b529eadbd72bed196adfbc41368b2d50493efce6e716320"
+                    "type": "git",
+                    "url": "https://github.com/FFMS/ffms2.git",
+                    "branch": "master",
```

Comment:
```suggestion
```

never a branch.

Reply:
if that version works with ffmpeg 5 that is in the SDK maybe building ffmpeg can be avoided.

Reply:
OK, but if I can't use a branch, how I can do?

Reply:
just use the commit.